packages/components/src/toggle-group-control/toggle-group-control/component.tsx

```diff
@@ -47,6 +47,8 @@ function UnconnectedToggleGroupControl(
 	} = useContextSystem( props, 'ToggleGroupControl' );
 
 	const baseId = useInstanceId( ToggleGroupControl, 'toggle-group-control' );
+	const normalizedSize =
+		__next40pxDefaultSize && size === 'default' ? '__unstable-large' : size;
```

Comment:
We usually don't do this kind of preemptive size normalization in other components, but in this specific case it was a lot cleaner this way. Otherwise we'd have to drill the prop down to multiple subcomponents, touching a handful of more files.

Reply:
Agreed. We should be able to clean this up after the migration is over and we remove temporary props and `size` values